feature-libs/product-multi-dimensional/selector/components/guards/product-multi-dimensional-selector.guard.spec.ts

```diff
@@ -65,7 +65,7 @@ class MockSemanticPathService implements Partial<SemanticPathService> {
   }
 }
 
-describe('ProductMultiDimensionalGuard', () => {
+xdescribe('ProductMultiDimensionalGuard', () => {
```

Comment:
`major:` let's remove `x`prefix

Reply:
sure I can remove. These are old tests and much has changed in that file. I've already fixed them here  https://github.com/SAP/spartacus/pull/19118